bundles/io/org.eclipse.smarthome.io.audio/src/main/java/org/eclipse/smarthome/io/audio/AudioCodec.java

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
Please use the standard license header by running `mvn license:format` on the root of the project.
We maintain the list of authors (and thus their copyright) through the author tags.